image-rs/src/pull.rs

```diff
@@ -19,19 +16,13 @@ use crate::decrypt::Decryptor;
 use crate::image::LayerMeta;
 use crate::meta_store::MetaStore;
 use crate::stream::stream_processing;
-use crate::unpack::unpack;
-
-const DIGEST_SHA256: &str = "sha256";
-const DIGEST_SHA512: &str = "sha512";
 
 const ERR_NO_DECRYPT_CFG: &str = "decrypt_config is None";
-const ERR_BAD_UNCOMPRESSED_DIGEST: &str = "unsupported uncompressed digest format";
-const ERR_BAD_COMPRESSED_DIGEST: &str = "unsupported compressed digest format";
```

Comment:
These two const str still needed by the `test_async_handle_layer`

Reply:
It would be better use a single definition instead of using two from both stream.rs and pull.rs.
So moved `ERR_BAD_UNCOMPRESSED_DIGEST` into lib.rs